code/controllers/configuration/entries/game_options.dm

```diff
@@ -105,6 +105,31 @@
 	config_entry_value = 2
 	min_val = 0
 
+/datum/config_entry/number/incursion_cost_base	//how many players per the first incursionist
+	config_entry_value = 6
+	integer = FALSE
+	min_val = 0
+
+/datum/config_entry/number/incursion_cost_increment	//How much each member of the incursion's spawn price increases by
+	config_entry_value = 2
+	integer = FALSE
+	min_val = 0
+
+/datum/config_entry/number/incursion_count_min	//Min number of players in the group
+	config_entry_value = 6
+	integer = TRUE
+	min_val = 1
+
+/datum/config_entry/number/incursion_count_max	//Max number of players in the group
+	config_entry_value = 6
+	integer = TRUE
+	min_val = 1

```

Comment:
You sure you want min and max to both be 6?

Reply:
whoops